slopmop/checks/general/_web_meta.py

```
@@ -0,0 +1,160 @@
+its own URL. This module holds that parsing using only the standard library
+(``html.parser`` plus a flat regex for the sitemap), so the gate stays
+``ToolContext.PURE`` with no extra deps and no XML-parser attack surface.
+"""
+
+from __future__ import annotations
+
+import re
+from dataclasses import dataclass
+from html.parser import HTMLParser
+from pathlib import Path
+from typing import List, Optional, Tuple
```

Ruff (0.15.17)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)